images/linux/ubuntu1604.json

```diff
@@ -243,7 +243,8 @@
                 "{{template_dir}}/scripts/installers/nginx.sh",
                 "{{template_dir}}/scripts/installers/android.sh",
                 "{{template_dir}}/scripts/installers/pypy.sh",
-                "{{template_dir}}/scripts/installers/python.sh"
+                "{{template_dir}}/scripts/installers/python.sh",
+                "{{template_dir}}/scripts/installers/kotlin.sh"
```

Comment:
we don't need to add anything to Ubuntu 16 as it's being deprecated
https://github.com/actions/virtual-environments/issues/3287